interpreter/cling/include/cling/MetaProcessor/MetaLexer.h

```diff
@@ -70,6 +74,13 @@ namespace cling {
 
     bool isNot(tok::TokenKind K) const { return kind != K; }
     bool is(tok::TokenKind K) const { return kind == K; }
+    bool isClosingBrace() const
+    { return kind == tok::r_square || kind == tok::r_paren
+        || kind == tok::r_brace; }
+    ///\brief Return whether this instance matches a opening brace `K`; assumes
+    /// that `r_xxx` brace follows `l_xxx` on the TokenKind enumeration.
+    bool closesBrace(tok::TokenKind K) const
+    { return isClosingBrace() && (kind == K+1); }
```

Comment:
The indentation seems awkward. Can you split it in multiple lines?

Reply:
Fixed.